bootstrap_cfn/config.py

```diff
@@ -544,7 +552,20 @@ def elb(self):
                 )
                 load_balancer.SecurityGroups = [Ref(sg)]
                 elb_sgs.append(sg)
-        return elb_list, elb_sgs
+
+            # Add outputs
+            output_name = "ELB" + safe_name
+            logging.debug("config:elb:Adding output to ELB '%s'" % (output_name))
+            template.add_output(Output(
+                output_name,
+                Description="ELB DNSName",
+                Value=GetAtt(load_balancer, "DNSName")
+            ))
+
+        # Update template with ELB resources
+        map(template.add_resource, elb_list)
+        map(template.add_resource, elb_sgs)
+        template = self._attach_elbs(template)
```

Comment:
I guess it's a bit difficult in this instance but I don't really like us not returning something here.